### ISSUE TITLE
Doesn't support workspaces only on primary

### ISSUE THREAD

**Issue by wtheisen** [paperwm/PaperWM#216]
When running on multiple monitors the second monitor appears not to respect the tiling aspect of the software and continues to act like a normal gnome desktop session. This is with extend workspaces. Additionally many windows seem to "pop out" of the tiling system and then refuse to be put back into it.

Great software on my laptop with a single screen but it's been giving me some problems on dual monitors.

**Comment by hedning** [comment]
Yeah, we don't support «workspaces on display only» (ie. `org.gnome.mutter workspaces-only-on-primary true`). Switching to «Workspaces span displays»  should fix it (or set `workspaces-only-on-primary` to false in `dconf-editor`).

We should probably make this more prominent in the readme though.

**Comment by cben** [comment]
FWIW, I'm benefitting from the current lack of support :grin:
My use case is quite esoteric with monitors showing *overlapping* areas of the video memory.
When I open my laptop to use its webcam in meetings, it obscures part of my (much larger) external monitor.  A workaround I use is to make the laptop "transparent" by mirroring the area it obscures :exploding_head:, using a command like:
```
xrandr --output eDP-1 --fb 3840x2160 --scale 0.74x0.74 --pos 1139x1577
```
But with «Workspaces span displays», PaperWM was thinking it has 2 separate monitors, and windows from another workspace were jumping on top of the laptop monitor, it was unusable.  With «workspaces on primary display only» it works fine as one monitor :laughing: 

![IMG_20200719_120548](https://user-images.githubusercontent.com/273688/87871376-7dcd6680-c9b8-11ea-90b2-a6e02f91ecea.jpg)

Another sillier workaround I tend to use is add a dummy terminal window below some window to make it fit above the laptop screen...

In an ideal world:
- there would be an option to "hide" specified monitors from PaperWM.
- there would be a way to exclude a top/bottom portion of a monitor from window placement. (surpisingly, this is nearly impossible with xrandr!)

but of course feel free to fix this :+1:  I can poke around the code if necessary for my needs.

**Comment by Thesola10** [comment]
Superseded by #932 for visibility -- PaperWM now sets workspaces to span displays out of the box.

---

_🧹 Just going through and triaging old issues..._
